python_modules/dagster/dagster/core/asset_defs/assets.py

```diff
@@ -396,9 +399,29 @@ def get_resource_requirements(self) -> Iterator[ResourceRequirement]:
     def required_resource_keys(self) -> Set[str]:
         return {requirement.key for requirement in self.get_resource_requirements()}
 
+    def to_string(self):
+        if len(self.asset_keys) == 1:
+            return f"AssetsDefinition with key {self.asset_key.to_string()}"
+        else:
+            asset_keys = ", ".join(
```

Comment:
tiny nit: for dbt assets (and, presumably, others) the list of asset keys can get really large (>100). Maybe just show the first 5 and then "..."?

Doesn't need to be in this pr.

Reply:
Ugh yea good point.